server/graphql/common/transactions.ts

```diff
@@ -89,7 +99,7 @@ export const canRefund = async (transaction, _, req): Promise<boolean> => {
   if (transaction.type !== TransactionTypes.CREDIT || transaction.OrderId === null) {
     return false;
   }
-  return remoteUserMeetsOneCondition(req, transaction, [isRoot, isPayeeHostAdmin]);
+  return remoteUserMeetsOneCondition(req, transaction, [isRoot, isPayeeHostAdmin, isPayeeCollectiveAdmin]);
```

Comment:
I'm surprised by the wording here `payee`, it's something that I expect for Expenses and not for Contributions. No need to take action now but just wanted to mention that. cc @Betree

Reply:
@kewitz was the one introducing the term in this file. I stayed consistent when I added resolvers here because even though we don't use it in other places, the term `payee` seemed pretty appropriate to me - it's the person who receives the money from the transaction (while a `toAccount` can have different meanings if it's a CREDIT or DEBIT).

Reply:
The logic is quite simple, it is just the composition between transaction type and collective. The problem is that we're abstracting that into a function if we want to keep consistency and the function will require a name.
I also considered `beneficiary` at the time, but since we already have payee elsewhere, it seemed simpler.